engine/src/dskw32.cpp

```diff
@@ -1617,7 +1617,7 @@ struct MCWindowsDesktop: public MCSystemInterface, public MCWindowsSystemService
     
 	virtual bool GetVersion(MCStringRef& r_string)
     {
-        return MCStringFormat(r_string, "NT %d.%d", (MCmajorosversion >> 8) & 0xFF, MCmajorosversion & 0xFF);
+        return MCStringFormat(r_string, "NT %d.%d", MCOSVersionGetMajor(MCmajorosversion), MCOSVersionGetMinor(MCmajorosversion));
     }
```

Comment:
Shouldn't that be:

`return MCStringFormat(r_string, "NT %d.%d", MCOSVersionGetMinor(MCmajorosversion), MCOSVersionGetBugfix(MCmajorosversion));`